item-api/src/main/java/com/example/itemapi/domain/model/Item.java

```diff
@@ -24,12 +24,14 @@ public class Item {
     private String name;
 
     @Embedded
-    @AttributeOverride(name = "price", column = @Column(name = "price", nullable = false))
     private Price price;
 
     @Column(name = "stock", nullable = false)
     private int stock;
 
+    @Column(name = "delete_yn", nullable = false)
+    private boolean deleteYn;
```

Comment:
```suggestion
    private boolean deleteYn;
```

`deleteYn` 필드는 예상되는게 `Y/N` 으로 문자열이 먼저 떠오르긴하네요!

`isDeleted` 나  `String deleteYn` 은 어떨까요?

Reply:
넵! isDeleted 수정 하겠습니다~!